packages/nimbus/src/components/card/card.mdx

```diff
@@ -10,201 +10,394 @@ menu:
-          <Card.Header>
-            <b>Background: {bg}</b>
-          </Card.Header>
-          <Card.Content>
-            <p>This card demonstrates the {bg} background style.</p>
-          </Card.Content>
-        </Card.Root>
-      ))}
-    </Stack>
-  )
-}
+const App = () => (
+  <Card.Root cardPadding="md" borderStyle="outlined" width="384px">
+    <Card.Content>
+      <Stack direction="horizontal" gap="400">
+        <Box
+          color="primary.11"
+          boxSize="1000"
+          borderRadius="50%"
+          bg="primary.3"
+          display="flex"
+          alignItems="center"
+          justifyContent="center"
+          flexShrink="0"
+        >
+          <Icons.ViewInAr width="24px" height="24px" />
+        </Box>
+        <Stack gap="100">
+          <Text fontWeight="bold">
+            Learn more about products
+          </Text>
+          <Text color="neutral.11">
+            Discover what a product list is and how to manage it.
+          </Text>
+        </Stack>
+      </Stack>
+    </Card.Content>
+  </Card.Root>
+);
 ```
 
-### Combined Configurations
-
-Cards can combine various styling options to achieve the desired visual presentation.
+> [!CAUTION]\
+> **Don't**
+>
+> - Don't use too many actions for the user within cards.
+> - Cards are generally not suitable for primary site navigation.
 
 ```jsx-live
 const App = () => (
-  <Stack gap="400">
-    <Card.Root cardPadding="md" borderStyle="outlined" elevation="none" backgroundStyle="default">
-      <Card.Header>
-        <b>Standard Card</b>
-      </Card.Header>
+  <Stack gap="400" alignItems="start">
+    {/* Main Card */}
+    <Card.Root cardPadding="md" borderStyle="outlined" width="423px">
       <Card.Content>
-        <p>A standard card with outline and default background.</p>
-      </Card.Content>
-    </Card.Root>
-    
-    <Card.Root cardPadding="lg" borderStyle="none" elevation="elevated" backgroundStyle="muted">
-      <Card.Header>
-        <b>Elevated Card</b>
-      </Card.Header>
-      <Card.Content>
-        <p>An elevated card with no border and muted background.</p>
+        <Stack direction="horizontal" gap="400">
+          <Box
+            alignSelf="flex-start"
+            color="primary.11"
+            boxSize="1000"
+            borderRadius="50%"
+            bg="primary.3"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            flexShrink="0"
+          >
+            <Icons.ViewInAr width="24px" height="24px" />
+          </Box>
+          <Stack direction="horizontal" gap="400" alignItems="center">
+            <Stack gap="200">
+              <Flex gap="100" alignItems="center">
+                <Text fontWeight="bold" color="primary.11">
+                  See your products
+                </Text>
+                <Box color="primary.11" boxSize="600">
+                  <Icons.ArrowForward width="full" height="full" />
+                </Box>
```

Comment:
@ByronDWall @misama-ct I really do feel we need to discuss how we are asking consumers to style Icons in Nimbus. Wrapping a box, applying styles there assuming the Icon inherits said styling, and then having to match height and width internally is clumsy IMO, unless I am still missing something. I feel icons should take these properties directly 🤔

Reply:
many design systems have an `Icon` component, where you pass the name of the icon you want to display as a prop. eg [what this article suggests](https://dev.to/seanyasno/handling-icons-in-react-best-practices-22c5)  We will probably end up implementing either something like that, or some other component/icon helper.

Im not as clear on how much of a priority that is, however